regress/expected/cypher_match.out

```diff
@@ -2310,13 +2310,31 @@ SELECT * FROM cypher('cypher_match', $$ MATCH () MATCH (_{name: "Dave"}) RETURN
  a 
 ---
  0
-(1 row)
+ 0
+ 0
+ 0
+ 0
+ 0
+ 0
+ 0
+ 0
+ 0
+(10 rows)
 
```

Comment:
This one needs to be looked into... previously `1 row` but now `10 rows`

Reply:
Yes, please try on Neo4j to see what it outputs :)

Reply:
> Yes, please try on Neo4j to see what it outputs :)

Yes.. already doing that